cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -58,6 +60,7 @@ class CliTest {
         Cli(
             valueFactory,
             input?.byteInputStream(Charsets.UTF_8) ?: EmptyInputStream(),
+            InputFormat.ION,
```

Comment:
Could have some tests with `InputFormat.PARTIQL`. Also I don't think you should hard-code the this argument to the `Cli` but rather pass it in as another argument to `makeCli`. You could set the default `inputFormat` for `makeCli` to `ION` depending on what the actual default is.